src/Presentation/Startup.cs

```diff
@@ -1,21 +1,36 @@
 using System;
+using Data;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
 
 namespace Presentation
 {
+    
     public class Startup
     {
+        
+        public string ConnectionString { get; set; }
         private readonly IConfiguration _config;
         
-        public Startup(IConfiguration config) =>_config = config;
+        public Startup(IConfiguration configuration)
+        {
+            _config = configuration;
+            Configuration = configuration;
+            ConnectionString = Configuration.GetConnectionString("DefaultConnection");
+        }
         
-        public static void ConfigureServices(IServiceCollection services)
+       
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
         {
+            ConnectionString = Configuration.GetConnectionString("DefaultConnection");
```

Comment:
Should we try to fetch the value from inside `Settings/settings.default.json` instead? @danidoff :pray: